src/server/user.ts

```diff
@@ -1,10 +1,8 @@
-import { RamadAds } from "../types/User";
+import { RamadAd, RamadAdsSQL } from "../types/User";
```

Comment:
`RamadAdSQL` maybe? 🤔 

Just like you did with `RamadAd` vs `RamadAds`

Reply:
Done